openquake/hazardlib/valid.py

```diff
@@ -664,9 +664,7 @@ def intensity_measure_types(value):
       ...
     ValueError: Duplicated IMTs in SA(0.1), SA(0.10)
     >>> intensity_measure_types('SA(1), PGA')
-    Traceback (most recent call last):
-    ...
-    ValueError: The IMTs are not sorted by period: SA(1), PGA
+    ['PGA', 'SA(1.0)']
```

Comment:
Could add 'PGV' here to illustrate how it will get sorted wrt the 'PGA' and 'SA' strings.